cmd/build.go

```diff
@@ -348,6 +350,9 @@ func buildNormal(targetOS string, vmArguments []string) {
 	if buildCachePath == config.BuildCachePathDefault && config.GetConfig().CachePath != "" {
 		buildCachePath = config.GetConfig().CachePath
 	}
+	if buildEngineVersion == config.BuildEngineDefault && config.GetConfig().Engine != "" {
+		buildEngineVersion = config.GetConfig().Engine
+	}
```

Comment:
Add a warning that there could be errors if the engine version doesn't match with the one used by flutter.